tests/commit/Unit_Test_U_net.py

```diff
@@ -0,0 +1,69 @@
+# import os; os.environ['CUDA_VISIBLE_DEVICES'] = ""
+from unittest import TestCase
+
+from phi import math
+from phi.math import channel, batch
+from phi.tf import nets as tf_nets
+from phi.torch import nets as torch_nets
+from phi.jax.stax import nets as stax_nets
+
+
+LIBRARIES = [tf_nets, torch_nets, stax_nets]
+
+
+
+class TestNetworks(TestCase):
+
+    def test_u_net_2d_network_sizes(self):
+        for lib in LIBRARIES:
+            net = lib.u_net(2, 3, batch_norm=False, activation='SiLU', in_spatial=2, use_res_blocks=True)
+            self.assertEqual(39059, lib.parameter_count(net))
+
+    def test_u_net_3d_norm_network_sizes(self):
+        for lib in LIBRARIES:
+            net = lib.u_net(2, 3, batch_norm=True, activation='SiLU', in_spatial=3, use_res_blocks=True)
+            self.assertEqual(113939, lib.parameter_count(net))
+
+    def test_u_net_1d_norm_network_sizes(self):
+        for lib in LIBRARIES:
+            net = lib.u_net(2, 3, batch_norm=True, activation='SiLU', in_spatial=1, use_res_blocks=True)
+            self.assertEqual(14867, lib.parameter_count(net))
+
+    def test_optimize_u_net(self):
+        for lib in LIBRARIES:
+            net = lib.u_net(1, 1, levels=2, use_res_blocks=True, activation='SiLU')
+            optimizer = lib.adam(net, 1e-3)
+
+            def loss_function(x):
+                print("Running loss_function")
+                assert isinstance(x, math.Tensor)
+                pred = math.native_call(net, x)
+                return math.l2_loss(pred)
+
+            for i in range(2):
+                lib.update_weights(net, optimizer, loss_function, math.random_uniform(math.batch(batch=10), math.spatial(x=8, y=8)))
+
+    def test_optimize_u_net_jit(self):
+        for lib in LIBRARIES:
+            net = lib.u_net(1, 1, levels=2, use_res_blocks=True, activation='SiLU')
+            optimizer = lib.adam(net, 1e-3)
+
+            @math.jit_compile
+            def loss_function(x):
+                print("Tracing loss_function")
+                assert isinstance(x, math.Tensor)
+                pred = math.native_call(net, x)
+                return math.l2_loss(pred)
+
+            for i in range(2):
+                lib.update_weights(net, optimizer, loss_function, math.random_uniform(math.batch(batch=10), math.spatial(x=8, y=8)))
+
+
+test_obj = TestNetworks()
```

Comment:
These lines are not necessary. The tests will be automatically run by `pytest`

Reply:
I merged the functions from Unit_Test_U_net.py to existing test_networks.py file. I hope I didn't break anything.

Reply:
Okay, then `Unit_Test_U_net.py` can now be removed, right?

Reply:
Yes. I have removed it from the commit/tests now.